changelogs/fragments/5829-fix-yarn-global.yml

```diff
@@ -0,0 +1,4 @@
+bugfixes:
+  - yarn - fix ``state=latest`` not working with ``global=true`` (https://github.com/ansible-collections/community.general/pull/5829).
```

Comment:
```suggestion
  - yarn - fix ``state=latest`` not working with ``global=true`` (https://github.com/ansible-collections/community.general/issues/5712).
```

Reply:
When fixing a reported bug, please refer to it. Alternatively, you may list both of them separated by comma - there is no specific norm about doing one way or the other.